MekHQ/src/mekhq/campaign/parts/MissingBattleArmorSuit.java

```diff
@@ -260,7 +260,7 @@ public void fix() {
         			origArmor.updateConditionFromEntity(false);
         		} else if(childPart instanceof BattleArmorEquipmentPart) {
         			for(MissingBattleArmorEquipmentPart p : missingStuff) {
-	            		if(null != childPart && null != p.getUnit() && p.isAcceptableReplacement(childPart, false)) {
+	            		if(null != p.getUnit() && p.isAcceptableReplacement(childPart, false)) {
```

Comment:
I don't see why this childPart null check was removed - it doesn't seem like it's redundant.

Reply:
If `childPart` is an instance of `BattleArmorEquipmentPart` it cannot be null.